homework_8/geometricFigure/GeometricFigure.java

```diff
@@ -0,0 +1,13 @@
+package geometricFigure;
+
+public interface GeometricFigure {
+    double getArea();
+
+    static void summAreas(GeometricFigure[] geometricFigures) {
+        double areaSumm = 0;
+        for (GeometricFigure figure : geometricFigures) {
+            areaSumm += figure.getArea();
+        }
+        System.out.println("Area of Circle + Triangle + Square = " + areaSumm);
```

Comment:
methods should not contain any print statements! Move it to the `main` method in `FigureMain` class